pino.js

```diff
@@ -258,6 +259,7 @@ function pino (opts, stream) {
   // internal options
   iopts.stringify = iopts.safe ? stringifySafe : JSON.stringify
   iopts.formatOpts = {lowres: true}
+  iopts.messageKeyString = ',' + JSON.stringify('' + iopts.messageKey) + ':'
```

Comment:
Actually it looks like there's a string template in lib/tools.js, so I guess that's a green light - I prefer the string template solution so I'll make that change and then we should be good to go